pwshlib.md

```diff
@@ -0,0 +1,86 @@
+# pwshlib
+
+This module enables you to leverage the `ruby-pwsh` gem to execute PowerShell from within your Puppet providers without having to instantiate and tear down a PowerShell process for each command called.
```

Comment:
The forge will still show the regular README.md

Reply:
Yeah, I had a think about this last night, this is one of the problems with having it all in one repo.

I think a build task could rejigger things long enough to do a pdk build, then put them back, but I don't think rewriting the main readme to make more sense on the forge is the right move.